Cargo.toml

```diff
@@ -29,10 +29,10 @@ saphir = { version = "0.9.5", features = ["https", "request_handler"] }
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
-slog = { version = "2.5", features = ["max_level_trace","release_max_level_trace"] }
+slog = { version = "2.5", features = ["max_level_trace","release_max_level_debug"] }
```

Comment:
Does it change something on the performance ? Or on the executable size ?  I don't think we have already logged with trace level in production, usually, debug should be enough, but what do we win to change it to debug ? Thanks

Reply:
Oh, I had not seen your message in the PR.  Ok, if performance is better, I'm OK with that.